contracts/mainnet/common/interfaces.sol

```diff
@@ -11,6 +11,7 @@ interface TokenInterface {
     function balanceOf(address) external view returns (uint);
     function decimals() external view returns (uint);
     function totalSupply() external view returns (uint);
+    function allowance(address owner, address spender) external view returns (uint256);
```

Comment:
@Richa-iitr Remove this.

Reply:
```suggestion
```

Reply:
Should i move it to compound interface? Since I am using it to get amount for manager functions.